modules/osint/username_search.py

```diff
@@ -37,7 +37,7 @@
 
 def thread(self, data, query,thread_count):
 	threadpool = concurrent.futures.ThreadPoolExecutor(max_workers=thread_count)
-	futures = (threadpool.submit(check, self, data[site]['url'].format(self.urlib(query).quote_plus(query).replace('.','%2E')), site, data) for site in data)
+	futures = (threadpool.submit(check, self, data[site]['url'].format(self.urlib(query).quote_plus.replace('.','%2E')), site, data) for site in data)
```

Comment:
`quote_plus` isn't defined in `core/urlib.py`, so it makes error. Made a corresponding method, might need revision